README.md

```diff
@@ -765,4 +758,76 @@ EXAMPLE
+```
+
+USAGE
+  $ sfdx sfpowerkit:profile:sync [-f <array>] [-n <array>] [-u <string>] [--apiversion <string>] [--json] [--loglevel trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]
+
+OPTIONS
+  -f, --folder=folder                             comma separated list of folders to scan for profiles. If ommited, all folders in the package directories will be used.
+  -n, --profilelist=profilelist                   comma separated list of profiles. If ommited, all the profiles found in the folder(s) will be synchronized
+  -u, --targetusername=targetusername             username or alias for the target org; overrides default target org
+  --apiversion=apiversion                         override the api version used for api requests made by this command
+  --json                                          format output as json
+  --loglevel=(trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL)  [default: warn] logging level for this command invocation
+
+EXAMPLES
+  $ sfdx sfpowerkit:source:profile:sync -u prod
+  $ sfdx sfpowerkit:source:profile:sync  -f force-app -n "My Profile" -r -u prod
+  $ sfdx sfpowerkit:source:profile:sync  -f "module1, module2, module3" -n "My Profile1, My profile2"  -u prod
+```
+_See code: [src\commands\sfpowerkit\profile\sync.ts](https://github.com/Accenture/sfpowerkit/blob/master/src/commands/sfpowerkit/profile/sync.ts)_
+
+
+## `sfpowerkit:profile:reconcile`
+
+cleanup profile configurations to make them compatible with the target org.
+```
+
+USAGE
+  $ sfdx sfpowerkit:profile:reconcile [-f <array>] [-n <array>] [-u <string>] [--apiversion <string>] [--json] [--loglevel
+  trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]
+
+OPTIONS
+  -f, --folder=folder                               the folders to Scan. You can provide a comma separated list of folder. If ommited, the folders listed in the package directories will be used.
+  -n, --profilelist=profilelist                     the profile names that will be reconcile. if ommited, all the profiles components will be reconciled.
+  -u, --targetusername=targetusername               username or alias for the target org; overrides default target org
+  --apiversion=apiversion                           override the api version used for api requests made by this command
+  --json                                            format output as json
+  --loglevel=(trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL)  [default: warn] logging level for this command invocation
+
+EXAMPLES
+  $ sfdx sfpowerkit:profile:reconcile  --folder force-app
+  $ sfdx sfpowerkit:profile:reconcile  --folder force-app,module2,module3 -u sandbox
+  $ sfdx sfpowerkit:profile:reconcile  -u myscratchorg
+```
+
+## `sfpowerkit:profile:merge`
+
+retrieve profiles from a development environment and merge it with the profile configurations
```

Comment:
Can you provide more details on how this command actually works? what will be stripped of?

Reply:
This command do the same thing as the sync command but instead overwriting  the profile metadata files, it update it configuration with those retrieve from the org. This avoid loosing profiles settings from one org to another.

Running the sync from the production org and the merge from a scratch org will result in our profile metadata containing settings that exist in both orgs but also settings that are available only in each org. 

The reconcile is then necessary to clean up profile to make them compatible to deploy in any org by removing what is not supported in the org.

Reply:
Okie, understood, lets get the readme little elaborated